src/test/java/bytebot/command/TaskCommandTest.java

```diff
@@ -0,0 +1,111 @@
+package bytebot.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import org.junit.jupiter.api.Test;
+
+import bytebot.ByteException;
+import bytebot.storage.Storage;
+import bytebot.task.Task;
+import bytebot.task.TaskList;
+
+public class TaskCommandTest {
+
+    private Storage createStorage() {
+        return new Storage() {
+            {
+                initializeWithTaskList(new TaskList());
+            }
+
+            @Override
+            public void saveTasks(TaskList tasks) {
+            }
+
+            @Override
+            public void addTask(Task task) {
+                getTaskList().add(task);
+            }
+        };
+    }
+
+    @Test
+    public void todoCommand_addsTaskAndNotifiesUi() throws Exception {
+        Storage storage = createStorage();
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        PrintStream original = System.out;
+        System.setOut(new PrintStream(baos));
+        try {
+            new TodoCommand("read book").execute(new bytebot.ui.Ui(), storage);
+        } catch (Exception e) {
+            fail("TodoCommand execution failed with exception: " + e.getMessage());
+        } finally {
+            System.setOut(original);
+        }
+        assertEquals(1, storage.getSize());
+        assertTrue(baos.toString().contains("I've added this task"));
+    }
+
+    @Test
+    public void todoCommand_emptyDescription() {
```

Comment:
Perhaps you could include the outcome of the test in the naming of the method?
```suggestion
    public void todoCommand_emptyDescription_printsErrorMessage() {
```